Introduction.md

```diff
@@ -123,12 +107,76 @@ This will allow you to run various tools you by name instead of typing the full
 
 ### Updating checkouts
 
-There is a script to update a WebKit checkout: `Tools/Scripts/update-webkit`. This script will automatically merge change logs mentioned below.
+There is a script to update a WebKit checkout: `Tools/Scripts/update-webkit`.
```

Comment:
Probably best to keep this for now, but I do think we should encourage folks to use raw `git` foo for this one.